backend/handlers/handlers.go

```diff
@@ -400,13 +400,13 @@ func (handlers *Handlers) postAddAccountHandler(r *http.Request) (interface{}, e
 		return nil, errp.WithStack(err)
 	}
 	// The following parameters only work for watch-only singlesig accounts at the moment.
-	jsonCoinCode := jsonBody["coinCode"]
+	jsonCoinCode := coin.Code(jsonBody["coinCode"])
 	jsonScriptType := jsonBody["scriptType"]
 	jsonAccountName := jsonBody["accountName"]
 	jsonExtendedPublicKey := jsonBody["extendedPublicKey"]
 	jsonAddress := jsonBody["address"]
 
-	coin, err := handlers.backend.Coin(jsonCoinCode)
+	theCoin, err := handlers.backend.Coin(jsonCoinCode)
```

Comment:
why did Go ever think it's a good idea to mix package namespace with variable namespace :(

Reply:
:smile: you used the `coinpkg` as a good workaround, I think. why not use it here as well?

Reply:
also, an old blog post about choosing package names: https://blog.golang.org/package-names

Reply:
done with `coinpkg`